### ISSUE TITLE
Creating Javadoc yields 4 warnings

### ISSUE THREAD

**Issue by ralphlange** [epics-base/pvDatabaseJava#1]
ssia

**Comment by mrkraimer** [comment]
I generated javadoc with eclipse and do not see any warning messages.
I also do not see any warning messages from
mvn package

What to do?

Marty

**Comment by ralphlange** [comment]
```
$ mvn install
[INFO] Scanning for projects...
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] Building pvDatabase - Java 4.2.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO]
[INFO] --- build-helper-maven-plugin:1.9.1:add-source (add-source) @ pvDatabaseJava ---
[INFO] Source directory: C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\example added.
[INFO] Source directory: C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\perform added.
[INFO] Source directory: C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\shell added.
...
[INFO]
[INFO] --- maven-javadoc-plugin:2.10.1:jar (attach-javadocs) @ pvDatabaseJava ---
[INFO]
Loading source files for package org.epics.pvdatabase.pva...
Loading source files for package org.epics.pvdatabase...
Loading source files for package org.epics.pvdatabase.example...
Loading source files for package org.epics.pvdatabase.pva.example...
Loading source files for package org.epics.pvdatabase.perform...
Constructing Javadoc information...
...
4 warnings
[WARNING] Javadoc Warnings
[WARNING] C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\src\org\epics\pvdatabase\PVRecord.java:195: warning: no @return
[WARNING] public final boolean addPVRecordClient(PVRecordClient pvRecordClient) {
[WARNING] ^
[WARNING] C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\src\org\epics\pvdatabase\PVRecord.java:214: warning: no @return
[WARNING] public final boolean removePVRecordClient(PVRecordClient pvRecordClient) {
[WARNING] ^
[WARNING] C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\src\org\epics\pvdatabase\PVRecord.java:251: warning: no @return
[WARNING] public final boolean addListener(PVListener listener,PVCopy pvCopy) {
[WARNING] ^
[WARNING] C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\src\org\epics\pvdatabase\PVRecord.java:285: warning: no @return
[WARNING] public final boolean removeListener(PVListener listener,PVCopy pvCopy) {
[WARNING] ^
[INFO] Building jar: C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\target\pvDatabaseJava-4.2.0-SNAPSHOT-javadoc.jar
[INFO]
[INFO] --- maven-install-plugin:2.3.1:install (default-install) @ pvDatabaseJava ---
[INFO] Installing C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\target\pvDatabaseJava-4.2.0-SNAPSHOT.jar to C:\Users\langer\.m2\repository\org\epics\pvDatabaseJava\4.2.0-SNAPSHOT\pvDatabaseJava-4.2.0-SNAPSHOT.jar
[INFO] Installing C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\pom.xml to C:\Users\langer\.m2\repository\org\epics\pvDatabaseJava\4.2.0-SNAPSHOT\pvDatabaseJava-4.2.0-SNAPSHOT.pom
[INFO] Installing C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\target\pvDatabaseJava-4.2.0-SNAPSHOT-sources.jar to C:\Users\langer\.m2\repository\org\epics\pvDatabaseJava\4.2.0-SNAPSHOT\pvDatabaseJava-4.2.0-SNAPSHOT-sources.jar
[INFO] Installing C:\Users\langer\Documents\Work\EPICS\V4\pvDatabaseJava\target\pvDatabaseJava-4.2.0-SNAPSHOT-javadoc.jar to C:\Users\langer\.m2\repository\org\epics\pvDatabaseJava\4.2.0-SNAPSHOT\pvDatabaseJava-4.2.0-SNAPSHOT-javadoc.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 17.749s
[INFO] Finished at: Wed Sep 23 14:00:38 CEST 2015
[INFO] Final Memory: 21M/218M
[INFO] ------------------------------------------------------------------------
```

**Comment by mrkraimer** [comment]
As far as I know the warnings no longer appear.

**Comment by ralphlange** [comment]
This is true.

Note, however, that you can close issues via commit messages. This links the issue to the commit that fixed it (and vice versa), thus improving traceability.

Something like "fixes #1" would have done the job.
Note that the issue is closed when the commit gets merged into the default branch - so it is safe to do that on release and development branches, private clones and pull requests.
